### PR TITLE
Update LM and UMAPs

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Update LM and UMAPs

In this PR, Plate 5 UMAP and LMs are added. The next steps after this PR are to create a CSV of LM coefficients from combined plate data (Plates 5, 3, and 3 prime) and UMAP with Plates 5, 3, and 3 prime together.

Question for @gwaybio:

For the LM coefficients of the combined data, will I need to combine the normalized data and then perform feature selection with all plates together before generating the LM coefficients?

### CODE REVIEW

**Comment by gwaybio** [comment]
> For the LM coefficients of the combined data, will I need to combine the normalized data and then perform feature selection with all plates together before generating the LM coefficients?

No, since this is a univariate test (modeling one feature at a time) it is ok to apply LM on the normalized data.

I'll review now!

File: 4.analyze_data/notebooks/UMAP/0.UMAP_coordinates.ipynb [hunk 1]
```diff
@@ -59,20 +59,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 5,
```
Comment (gwaybio) [inline 4.analyze_data/notebooks/UMAP/0.UMAP_coordinates.ipynb]: looks like this cell was executed out of order. (reminder to make sure cells are sequentially executed! 😄 )

**Comment by jenna-tomkinson** [comment]
Thank you for the answer I appreciate it! 😄

**Review by gwaybio — APPROVED** [review]
Looking forward to the next PR to see the interaction term!

File: 4.analyze_data/notebooks/UMAP/0.UMAP_coordinates.ipynb (hunk 1 above)
Comment (jenna-tomkinson) [reply 4.analyze_data/notebooks/UMAP/0.UMAP_coordinates.ipynb]: Ahh great catch! I'll fix prior to merging!

**Comment by jenna-tomkinson** [comment]
@gwaybio Thanks for the review! I am merging now!

**Pull request merged**